### PR TITLE
feat(qrm): Support for OOM priority as a QoS enhancement

### PR DESCRIPTION
**Pull request by y-ykcir**
#### What this PR does / why we need it:
related to this [issue](https://github.com/kubewharf/katalyst-core/issues/216)

### CODE REVIEW

File: pkg/consts/qos.go [hunk 1]
```diff
@@ -37,6 +37,8 @@ const (
 
 	PodAnnotationMemoryEnhancementNumaExclusive       = "numa_exclusive"
 	PodAnnotationMemoryEnhancementNumaExclusiveEnable = "true"
+
+	PodAnnotationMemoryEnhancementOOMPriority = "oom_priority"
```
Comment (csfldf) [inline pkg/consts/qos.go:41]: please add comments describing this enhancement and the valid values 


ps: other collaborators will also supplement comments for other enhancements

File: pkg/consts/qrm.go [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package consts
+
+// QRMPhase is the phase of each rpc call in qrm plugin lifecycle
```
Comment (csfldf) [inline pkg/consts/qrm.go:19]: remove lifecycle; "QRMPhase is the phase of each rpc call in qrm plugin" just ok

Comment (y-ykcir) [reply pkg/consts/qrm.go]: done

File: pkg/consts/qos.go (hunk 1 above)
Comment (y-ykcir) [reply pkg/consts/qos.go:55]: done

Comment (y-ykcir) [reply pkg/consts/qos.go:55]: done

**Review by csfldf — APPROVED** [review]

**Pull request merged**